external_search.py

```diff
@@ -2990,7 +2987,7 @@ def modify_search_query(self, search):
             search = search.update_from_dict(
                 dict(search_after=self.last_item_on_previous_page)
             )
-        return search
+        return super(SortKeyPagination, self).modify_search_query(search)
```

Comment:
To be safe, `SortKeyPagination` also needs to run regular pagination, just in case the server returned more results than we asked for.

Reply:
So, this will return `self.size` amount of results since `offset` is always 0?

Reply:
yep.